tests/filtering/test_require_corect_ionmode.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from matchms.filtering import require_correct_ionmode
+from ..builder_Spectrum import SpectrumBuilder
+
+
+@pytest.mark.parametrize("ionmode, ionmode_to_keep, spectrum_removed", [
+    ("positive", "positive", False),
+    ("negative", "negative", False),
+    ("positive", "both", False),
+    ("negative", "both", False),
+    ("positive", "negative", True),
+    ("negative", "positive", True),
+    ("n/a", "both", True),
+])
+def test_clean_compound_name(ionmode, ionmode_to_keep, spectrum_removed):
+    builder = SpectrumBuilder()
```

Comment:
I think this test should be renamed as it doesn't test cleaning the compound name?

Reply:
Yes! Copied an old test as scaffold...